monitoring/local_monitoring/kafka_demo/kafka_demo.sh

```diff
@@ -0,0 +1,416 @@
+}
+
+function setup_kafka_server() {
+  {
+  # Create namespace for Kafka if it doesn't exist
+  echo "Creating namespace for Kafka..."
+  oc create namespace $KAFKA_NAMESPACE || echo "Namespace $KAFKA_NAMESPACE already exists."
+
+  # Install Kafka using Strimzi Operator (if not already installed)
+  echo "Installing Strimzi Operator..."
+  oc apply -f https://strimzi.io/install/latest?namespace=$KAFKA_NAMESPACE -n $KAFKA_NAMESPACE
+
+  # Wait for the Strimzi Operator to be ready
+  echo "Waiting for Strimzi Operator to be ready..."
+  oc rollout status deployment/strimzi-cluster-operator -n $KAFKA_NAMESPACE
+
+  # Create Kafka cluster YAML
+  cat <<EOF | oc apply -n $KAFKA_NAMESPACE -f -
+  apiVersion: kafka.strimzi.io/v1beta2
+  kind: Kafka
+  metadata:
+    name: $KAFKA_CLUSTER_NAME
+  spec:
+    kafka:
+      version: 3.8.0
+      replicas: 3
+      listeners:
+        - name: plain
+          port: 9092
+          type: internal
+          tls: false
+        - name: tls
+          port: 9093
+          type: internal
+          tls: true
+        - name: external
+          port: 9094
+          type: route
+          tls: true
+      config:
+        offsets.topic.replication.factor: 3
+        transaction.state.log.replication.factor: 3
+        log.message.format.version: "3.4"
+      storage:
+        type: ephemeral
+    zookeeper:
+      replicas: 3
+      storage:
+        type: ephemeral
+    entityOperator:
+      topicOperator: {}
+      userOperator: {}
+EOF
+
+  # Wait for Kafka to be ready
+  echo "Waiting for Kafka cluster to be ready..."
+  oc wait kafka/$KAFKA_CLUSTER_NAME --for=condition=Ready --timeout=300s -n $KAFKA_NAMESPACE
+
+# Create Kafka topics
+echo "Creating Kafka topics..."
+cat <<EOF | oc apply -n $KAFKA_NAMESPACE -f -
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaTopic
+metadata:
+  name: recommendations-topic
+  labels:
+    strimzi.io/cluster: $KAFKA_CLUSTER_NAME
+spec:
+  partitions: 3
+  replicas: 3
+---
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaTopic
+metadata:
+  name: error-topic
+  labels:
+    strimzi.io/cluster: $KAFKA_CLUSTER_NAME
+spec:
+  partitions: 3
+  replicas: 3
+---
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaTopic
+metadata:
+  name: summary-topic
+  labels:
+    strimzi.io/cluster: $KAFKA_CLUSTER_NAME
+spec:
+  partitions: 3
+  replicas: 3
+EOF
+
+  # Get Kafka bootstrap server URL
+  BOOTSTRAP_SERVER="$KAFKA_CLUSTER_NAME-kafka-bootstrap.$KAFKA_NAMESPACE.svc.cluster.local:9092"
+
+  echo "✅ Kafka Bootstrap Server: $BOOTSTRAP_SERVER"
+  export BOOTSTRAP_SERVER
+  } >> "${LOG_FILE}" 2>&1
+}
+
+# Check if the cluster_type is one of icp or openshift
+function check_cluster_type() {
+	case "${CLUSTER_TYPE}" in openshift) ;;
+	*)
+		echo "Error: unsupported cluster type: ${CLUSTER_TYPE}"
+		echo "Currently only openshift cluster is supported"
+		exit -1
+		;;
+	esac
+}
+
+function consume_messages() {
+  echo -n "🔄 Consuming recommendations from the recommendations-topic..."
```

Comment:
Can you please include some dots to let user know that kafka is consuming messages and is in progress.